swift/codegen/templates/ql_class.mustache

```diff
@@ -8,6 +8,8 @@ class {{name}}Base extends {{db_id}}{{#bases}}, {{.}}{{/bases}} {
   {{#root}}
   string toString() { none() } // overridden by subclasses
 
+  string getPrimaryQlClass() { none() } // overridden by subclasses
```

Comment:
The predicate should actually be called `getAPrimaryQlClass`, and we should have a corresponding `getPrimaryQlClasses` defined as:
```ql
final string getPrimaryQlClasses() { result = concat(this.getAPrimaryQlClass(), ",") }
```
Currently, for Swift, we only return a single result. But that's not true for C/C++ today, and it's not necessarily the case in the future.
(See https://github.com/github/codeql/blob/main/cpp/ql/lib/semmle/code/cpp/Element.qll#L53 for how it's done for C/C++.)

Reply:
I see!